pages/dkp/konvoy/2.2/choose-infrastructure/aws/advanced/prerequisites/index.md

```diff
@@ -19,7 +19,7 @@ Before you begin using Konvoy, you must have:
 
 <p class="message--note"><strong>NOTE: </strong>On macOS, Docker runs in a virtual machine. Configure this virtual machine with at least 8GB of memory.</p>
 
-If you use these instructions to create a cluster on AWS using the DKP default settings without any edits to configuration files or additional flags, your cluster is deployed on an [Ubuntu 20.04 operating system image][supported-systems] with 3 control plane nodes, and 4 worker nodes.
+If you use these instructions to create a cluster on AWS using the DKP default settings without any edits to configuration files or additional flags, your cluster is deployed on an [Ubuntu 20.04 operating system image][supported-systems] with 3 control plane nodes, and 4 worker nodes. For Enterprise, the recommendation is at least four worker nodes. Essential, however, only requires one worker node.  DKP creates the worker nodes via CLI and will use acceptable numbers by default.
```

Comment:
This information came from Daniel.... The four worker nodes is a recommendation.
And those four nodes are just enough to run our applications.
For essential, we don't run anything on worker nods, so 1 worker should be a fine minimum.

Reply:
I'll be happy to make it say whatever it needs to say.  : )

Reply:
Ok, Craig agreed that I should add the pre-req from Pre-prov regarding Control Planes and Worker Nodes and then specify to customers what type of machine instances we default to rather than just the numbers.